ocaml/database/block_device_io.ml

```diff
@@ -476,11 +476,12 @@ let action_writedb block_dev_fd client datasock target_response_time =
     (* If there's space, write some ASCII NULs over the next few bytes so that we trample on any data which may already exist on the block device *)
     let remaining_space = remaining_space - min_space_needed in
     let trample_size = if size_size > remaining_space then remaining_space else size_size in
-    Unixext.time_limited_write block_dev_fd trample_size (String.make trample_size '\000') target_response_time;
+    Unixext.time_limited_write block_dev_fd trample_size (Bytes.make trample_size '\000') target_response_time;
```

Comment:
This could use `Unixext.time_limited_write_substring`

Reply:
`write_substring` uses a "safe" unsafe conversion (ans in the ocaml Unix module), I prefer to avoid it unless necessary